geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -170,6 +171,11 @@ Other configuration
 You also have to configure the `vars/srid`, `vars/alternate_projections`, `vars/resolutions` and
 `vars/extent` they will be dispatched using `c2ctemplate` or YAML link.
 
+Now we can esily switch from window / grid display query result by using `gmfQueryGrid` configuration
```

Comment:
"esily" -> "easily"
"from window / grid display query result" -> "between the display of query results in a window or in a grid"